api/v1beta2/ibmpowervsmachine_types.go

```diff
@@ -101,8 +100,8 @@ type IBMPowerVSMachineSpec struct {
 
 	// processors is the number of virtual processors in a virtual machine.
 	// when the processorType is selected as Dedicated the processors value cannot be fractional.
-	// maximum value for the Processors depends on the selected SystemType.
-	// when SystemType is set to e880 or e980 maximum Processors value is 143.
+	// maximum value for the Processors depends on the selected SystemType, which can be found here: https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-pricing-virtual-server-on-cloud
+	// when SystemType is set to e980 maximum Processors value is 143.
```

Comment:
Better to give value here for all the systypes or we can remove the two already given, so that user can lookup into the link IMO.